src/psyclone/tests/psyir/transformations/inline_trans_test.py

```diff
@@ -794,9 +960,10 @@ def test_validate_node():
 
 
 def test_validate_calls_find_routine(fortran_reader):
-    '''Test that validate() calls the _find_routine method.
-raises the expected error if the source of the
-    routine to be inlined cannot be found.'''
+    '''Test that validate() calls the _find_routine method raises the
```

Comment:
This docstring doesn't make a lot of sense (and I can see it didn't originally either - maybe a merge that didn't go well).

Reply:
Updated. It is nice that you tried to blame it on a merge :-)

Reply:
It was a genuine suggestion but it's nice to be nice :-)